content/components/mermaid-diagrams.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Mermaid diagrams"
+description: "Display diagrams using Mermaid"
+icon: "chart-network"
+---
+
+<RequestExample>
+
+````md Mermaid Flowchart Example
+```mermaid
+  flowchart LR
+    subgraph subgraph1
+        direction TB
+        top1[top] --> bottom1[bottom]
+    end
+    subgraph subgraph2
+        direction TB
+        top2[top] --> bottom2[bottom]
+    end
+    %% ^ These subgraphs are identical, except for the links to them:
+
+    %% Link *to* subgraph1: subgraph1 direction is maintained
+    outside --> subgraph1
+    %% Link *within* subgraph2:
+    %% subgraph2 inherits the direction of the top-level graph (LR)
+    outside ---> top2
+```
+````
+
+</RequestExample>
+
+[Mermaid](https://mermaid.js.org/) lets you create visual diagrams using text and code.
+
+You can create the following using Mermaid diagrams:
+
+- Flowchart
+- Sequence diagram
+- Class diagram
+- State diagram
+- Entity relationship diagram
+- User journey
+- and more
+
+For a complete list of diagrams supported by Mermaid, check out their [website](https://mermaid.js.org/).
+
+## Syntax for Mermaid diagrams
+
+To create a flowchart, you can write the Mermaid flowchart inside a Mermaid code block.
+
+````md
+```mermaid
+// Your mermaid code block here
+```
+````
+
+For example, the following code will render a Flowchart Mermaid diagram.
+
+````md
+```mermaid
+  flowchart LR
+    subgraph subgraph1
+        direction TB
+        top1[top] --> bottom1[bottom]
+    end
+    subgraph subgraph2
+        direction TB
+        top2[top] --> bottom2[bottom]
+    end
+    %% ^ These subgraphs are identical, except for the links to them:
+
+    %% Link *to* subgraph1: subgraph1 direction is maintained
+    outside --> subgraph1
+    %% Link *within* subgraph2:
+    %% subgraph2 inherits the direction of the top-level graph (LR)
+    outside ---> top2
+```
+````
+
+```mermaid
+  flowchart LR
+    subgraph subgraph1
+        direction TB
+        top1[top] --> bottom1[bottom]
+    end
+    subgraph subgraph2
+        direction TB
+        top2[top] --> bottom2[bottom]
+    end
+    %% ^ These subgraphs are identical, except for the links to them:
+
+    %% Link *to* subgraph1: subgraph1 direction is maintained
+    outside --> subgraph1
+    %% Link *within* subgraph2:
+    %% subgraph2 inherits the direction of the top-level graph (LR)
+    outside ---> top2
+```
```

Comment:
Oh I also mean having this (the generated example) at the top

Reply:
Done.

![image](https://github.com/mintlify/docs/assets/6391763/bed52a4a-c2df-4564-be01-acac4450f69d)